docs/8-kubernetes-container-orchestration/8.6-webhooks.md

```diff
@@ -98,14 +100,16 @@ Controller using python.
 
 2. Configure a validating webhook in `app/webhooks.py`. A reference is provided with `examples/warden-validating.py`.
 
+?> You might need to update out of date packages and OS versions in order for the webhook server to function properly
+
 #### Criteria
 
-- Only check pods in the `busybox` namespace.
-- If any container's image isn't `busybox:latest`, reject the pod.
+- Only check pods in the `myapp` namespace.
+- If any container's image isn't `mayapp:latest`, reject the pod.
```

Comment:
MAY app? @densellp

Reply:
That was fixed to "myapp" instead of "mayapp"